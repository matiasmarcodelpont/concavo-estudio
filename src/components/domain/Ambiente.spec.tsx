import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import { Ambiente } from './Ambiente'

const ambiente = {
  slug: 'elegance',
  name: 'Elegancee',
}

describe('Ambiente', () => {
  it('matches snapshot', () => {
    const { container } = render(<Ambiente {...ambiente} />)
    expect(container).toMatchSnapshot()
  })

  it('should render the ambiente name', () => {
    const { getByText } = render(<Ambiente {...ambiente} />)

    const descriptionElement = getByText(ambiente.name)
    expect(descriptionElement).toBeInTheDocument()
  })
})
