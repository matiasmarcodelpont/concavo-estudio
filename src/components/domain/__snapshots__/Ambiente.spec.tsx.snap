// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ambiente matches snapshot 1`] = `
<div>
  <a
    aria-label="Elegancee"
    class="relative w-full h-[61.8vw] max-h-[494px] pl-4 flex justify-center items-center"
    href="/casa-concavo/elegance"
  >
    <img
      alt="Elegancee"
      class="absolute inset-0 w-full h-full object-cover"
      data-nimg="fill"
      decoding="async"
      loading="lazy"
      sizes="100vw"
      src="/_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=3840&q=75"
      srcset="/_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=640&q=75 640w, /_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=750&q=75 750w, /_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=828&q=75 828w, /_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=1080&q=75 1080w, /_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=1200&q=75 1200w, /_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=1920&q=75 1920w, /_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=2048&q=75 2048w, /_next/image?url=%2Fambientes%2Felegance%2F0.jpeg&w=3840&q=75 3840w"
      style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
    />
    <div
      class="absolute inset-0 bg-black opacity-30 z-20"
    />
    <div
      class="relative max-w-[1600px] flex justify-between grow z-30"
    >
      <p
        class="text-xl sm:text-2xl md:text-3xl text-bone"
      >
        Elegancee
      </p>
      <svg
        class="lucide lucide-chevron-right size-[30px] sm:size-[40px] md:size-[40px]"
        fill="none"
        height="24"
        stroke="white"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="m9 18 6-6-6-6"
        />
      </svg>
    </div>
    <span
      style="font-size: 0px;"
    />
  </a>
</div>
`;
